### PR TITLE
[chore] Enbled make fix can fix table format

### PR DESCRIPTION
**Pull request by gyliu513**
Fixes https://github.com/open-telemetry/semantic-conventions/issues/1692

## Changes

Please provide a brief description of the changes here.

Note: if the PR is touching an area that is not listed in the [existing areas](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/README.md), or the area does not have sufficient [domain experts coverage](https://github.com/open-telemetry/semantic-conventions/blob/main/.github/CODEOWNERS), the PR might be tagged as [experts needed](https://github.com/open-telemetry/semantic-conventions/labels/experts%20needed) and move slowly until experts are identified.

## Merge requirement checklist

* [ ] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

File: docs/attributes-registry/android.md [hunk 1]
```diff
@@ -13,16 +13,16 @@
 
 The Android platform on which the Android application is running.
 
-| Attribute | Type | Description | Examples | Stability |
-|---|---|---|---|---|
+| Attribute                                                                            | Type   | Description                                                                                                                                                                                                                               | Examples   | Stability                                                        |
```
Comment (lmolkova) [inline docs/attributes-registry/android.md:16]: these files are auto-generated, we should not modify them (and I think whitespaces in jinja are too complicated to format them in this way)

**Review by lmolkova — COMMENTED** [review]
I believe we don't use format checks in the workflow and can probably remove this check from the makefile.

Wonder what @open-telemetry/specs-semconv-maintainers think

File: docs/attributes-registry/android.md (hunk 1 above)
Comment (gyliu513) [reply docs/attributes-registry/android.md:16]: @lmolkova `make fix-format` will update the tables

**Comment by gyliu513** [comment]
> I believe we don't use format checks in the workflows/CI and can probably remove this check from the makefile.

Yes, we can discuss if we need `make fix-format` or not, I have added `fix-format` to CI process in this PR.

**Comment by lmolkova** [comment]
> Yes, we can discuss if we need `make fix-format` or not, I have added `fix-format` to CI process in this PR.

as you can see it breaks other checks (registry validation). Personally, I don't see a benefit in format check in MD + yaml repo with MD and markdown lint checks.

**Comment by gyliu513** [comment]
Thanks @lmolkova , with that, shall we remove both `check-format` and `fix-format` from makefile? It is really annoying when running `make check`.

**Comment by lmolkova** [comment]
> Thanks @lmolkova , with that, shall we remove both `check-format` and `fix-format` from makefile? It is really annoying when running `make check`.

I don't use `make check` since it takes forever, so I wonder if other @open-telemetry/specs-semconv-maintainers have any thoughts.

**Comment by trask** [comment]
> I don't use `make check` since it takes forever, so I wonder if other @open-telemetry/specs-semconv-maintainers have any thoughts.

fwiw, the only thing I ever use is `make table-generation attribute-registry-generation`

**Comment by gyliu513** [comment]
Hi @lmolkova and @trask , I removed check-format and fix-format in Makefile, hope this is ok, thanks!

**Comment by joaopgrassi** [comment]
I think we discussed in some other PR/Issue that it would be good to start enforcing the line limit in markdown, but that's probably a bigger endeavor. For ex we would have to avoid modifying any part of the auto-generated content (tables etc). 

For now removing it sounds good to me. I also don't use it.

**Review by lmolkova — APPROVED** [review]

**Review by trask — APPROVED** [review]

**Pull request merged**